Get-help.md

```diff
@@ -48,9 +40,41 @@ If you have a question about a pull request or issue, you can also reach out to
 
 You can even mention whole teams of people! For example, if you find an issue that is destabilizing the project, you could notify all the core maintainers by including `@oppia/core-maintainers` in your issue. The teams are all listed [here](https://github.com/orgs/oppia/teams).
 
+### Email
+
+We have several mailing lists in the form of Google Groups that you can join(not much recommended):
```

Comment:
Please drop "(not much recommended)". Perhaps change to "...you can join, but please note that these are less active than GitHub Discussions:"